packages/core/contracts/oracle/implementation/Staker.sol

```diff
@@ -50,20 +53,31 @@ contract Staker is StakerInterface, Ownable, Testable {
     // Pulls tokens from users wallet and stakes them.
     function stake(uint256 amount) public override {
         _updateTrackers(msg.sender);
-        voterStakes[msg.sender].cumulativeStaked += amount;
-        cumulativeStaked += amount;
+        if (inActiveReveal()) {
```

Comment:
Why only during reveal and not commit too? Just seems a _little odd_ (although not necessarily a problem?) for balances to be able to change during the commit phase 🤷, but maybe that's fine to improve UX.

Reply:
for now this is just how the book keeping works...I'm going to work on removing this